Packet++/header/GvcpLayer.h

```diff
@@ -0,0 +1,632 @@
+#pragma once
+
+#include "IpAddress.h"
+#include "Layer.h"
+#include "MacAddress.h"
+#include "SystemUtils.h"
+#include <iostream>
+
+/**
+ * @file GvcpLayer.h
+ * @author An-Chi Liu (phy.tiger@gmail.com)
+ */
+
+/**
+ * @namespace pcpp
+ * @brief The main namespace for the PcapPlusPlus lib
+ */
+namespace pcpp
+{
+	namespace detail
+	{
+		static constexpr size_t kGvcpMagicNumber = 0x42;
+		static constexpr size_t kGvcpRequestHeaderLength = 8;
+		static constexpr size_t kGvcpAckHeaderLength = 8;
+		static constexpr size_t kGvcpDiscoveryBodyLength = 248;
+		static constexpr size_t kGvcpForceIpBodyLength = 56;
+	}  // namespace detail
+
+	typedef uint8_t GvcpFlag;  // flag bits are specified by each command
```

Comment:
nit: maybe use cpp11 `using` for alias declaration?

```suggestion
	using GvcpFlag = uint8_t;  // flag bits are specified by each command
```

Reply:
Fixed.